CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.0
+### Added
+* Agent: Added `job_id` and `task_id` to [configuration value expansion](docs/command-replacements.md) [#481](https://github.com/microsoft/onefuzz/pull/481)
+* Agent: Expanded the availablility of `tools_dir` to [configuration value expansion](docs/command-replacements.md) [#480](https://github.com/microsoft/onefuzz/pull/480)
+* Agent: Added clarifying context to command errors.  [#466](https://github.com/microsoft/onefuzz/pull/466)
+
+### Changed
+* CLI/Service/Agent: Supervisor can now be fully self-contained fuzzing tasks, no longer requiring `target_exe`.  Additionally, supervisor tasks can now optionally have managed report containers.  [#474](https://github.com/microsoft/onefuzz/pull/474)
+* Service: Managed nodes that are unused beyond 7 days are automatically reimaged to ensure OS patch levels are maintained.  [#476](https://github.com/microsoft/onefuzz/pull/476)
+* CLI/Service: Updated the default Windows VM image to `MicrosoftWindowsDesktop:Windows-10:20h2-pro:latest`.  Existing scalesets will not be impacted by this change, only newly created scalesets using the default image.  [#469](https://github.com/microsoft/onefuzz/pull/469)
+
+### Fixed
+* Agent: New inputs discovered by supervisor tasks are now saved to the `inputs` container.  [#484](https://github.com/microsoft/onefuzz/pull/484)
+* CLI: The license is now properly set in the python package metadata.  [#475](https://github.com/microsoft/onefuzz/pull/475)
+* Agent: Failure to download files via HTTP from queues now results in a failure, rater than the HTTP error being interpreted as the requested file.  [#485](https://github.com/microsoft/onefuzz/pull/485)
```

Comment:
```suggestion
* Agent: Failure to download files via HTTP from queues now results in a failure, rather than the HTTP error being interpreted as the requested file.  [#485](https://github.com/microsoft/onefuzz/pull/485)
```